libs/exavmlib/mix.exs.in

```diff
@@ -0,0 +1,75 @@
+#
```

Comment:
I'm not really sure why this is `.in` file, can't we make a real mix.exs file instead of templating it?

Reply:
This was included by mistake. It was from a work-in-progress to include the elixir documentation in our published API docs, which is not ready to PR yet. It has been removed.